### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ArcTanSusan**

### CODE REVIEW

File: testsuite/test_api.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def test_styleguide(self):
         self.reset()
 
     def test_styleguide_options(self):
-        # Instanciate a simple checker
+        # Instanstiate a simple checker
```
Comment (JelleZijlstra) [inline testsuite/test_api.py]: Instantiate

File: pycodestyle.py [hunk 2]
```diff
@@ -120,7 +120,7 @@
 LAMBDA_REGEX = re.compile(r'\blambda\b')
 HUNK_REGEX = re.compile(r'^@@ -\d+(?:,\d+)? \+(\d+)(?:,(\d+))? @@.*$')
 
-# Work around Python < 2.6 behaviour, which does not generate NL after
+# Work around Python < 2.6 behavior, which does not generate NL after
```
Comment (sigmavirus24) [inline pycodestyle.py]: This is actually a difference between American English and British English. I don't consider it a typo.

File: testsuite/test_api.py (hunk 1 above)
Comment (sigmavirus24) [inline testsuite/test_api.py]: Once you update this and revert the change to behaviour this will be good to go.

File: pycodestyle.py (hunk 2 above)
Comment (IanLee1521) [inline pycodestyle.py]: Agreed.
